lib/uber/inheritable_attr.rb

```diff
@@ -1,6 +1,7 @@
 module Uber
   module InheritableAttr
-    def inheritable_attr(name)
+
+    def inheritable_attr(name, options={})
       instance_eval %Q{
```

Comment:
You can simply use `options` in the `InheritableAttribute.inherit_for` call, that way we don't need any additional code.

```ruby
InheritableAttribute.inherit_for(self, :#{name}, options)
```

You don't need the `_cloneability` fix then! :grimacing: